docs/packaging-your-sql-project/introduction-to-the-dataset-and-problem.md

```diff
@@ -11,4 +11,42 @@ kernelspec:
   name: python3
 ---
 
-# Introduction to the dataset and problem: CO2 emissions of vehicles 
\ No newline at end of file
+# Introduction to the dataset and problem: CO2 emissions of vehicles
+
+To help consumers in Canada find fuel-efficient vehicles, the Government of Canada released a [fuel consumption ratings search tool](https://fcr-ccc.nrcan-rncan.gc.ca/en?_gl=1*1y4enqn*_ga*NzI2ODg1Njg2LjE2NjcyNDA3NTU.*_ga_C2N57Y7DX5*MTY2ODYzOTkyMy40LjAuMTY2ODYzOTkyMy4wLjAuMA). 
+
+In it, they provide users the ability to search vehicles by model, class and make and obtain information on the fuel consumption of various vehicles in three settings: city, highway and combined. Vehicles undergo 2-cycle and 5-cycle fuel consumption in each of these settings, and a co2 emissions score is assigned to the vehicle (for more information see [here](https://www.nrcan.gc.ca/energy-efficiency/transportation-alternative-fuels/fuel-consumption-guide/understanding-fuel-consumption-ratings/fuel-consumption-testing/21008) and here[https://www.nrcan.gc.ca/energy-efficiency/transportation-alternative-fuels/personal-vehicles/choosing-right-vehicle/buying-electric-vehicle/understanding-the-tables/21383]). 
+
+Additionally, they provide access through their [open data portal](https://open.canada.ca/data/en/dataset/98f1a129-f628-4ce4-b24d-6f16bf24dd64) as part of the [Open Government License Canada](https://open.canada.ca/en/open-government-licence-canada). 
+
+Whereas this tool allows consumers to obtain information via the website, it is desirable to develop an automated approach to extract information on the most recent models, and to be able to analyse the data to answer questions such as: 
+
+1. What is the average CO2 emissions of vehicles by class, make and model?
+2. Are there vehicles better suited for city driving than others (focus on less CO2 emissions)?
+3. Are there vehicles better suited for highway driving than others (focus on less CO2 emissions)?
+4. Are there vehicles better suited for combined driving than others (focus on less CO2 emissions)?
```

Comment:
I think points 2, 3, and 4 can be synthesized into a single point for eliminating some redundancy. I would recommend writing:
2. Which vehicles are better suited for different driving scenarios (city, highway, and combined) with a focus on lower CO2 emissions?

Reply:
Pushed fix in last commit